jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/bucketEditor/S3TreeTable.kt

```diff
@@ -65,4 +85,51 @@ open class S3TreeTable(private val treeTableModel: S3TreeTableModel) : TreeTable
     fun invalidateLevel(node: S3TreeNode) {
         node.parent?.removeAllChildren()
     }
+
+    private fun createDropTarget(): DropTarget {
+        val dropTarget = DropTarget()
+        try {
```

Comment:
The top level try is because `addDropTargetListener` can throw. Rearranged to use the `DropTarget` constructor instead so we don't have nested try catches anymore.